features/admin.actions.v1/pages/actions.tsx

```diff
@@ -0,0 +1,363 @@
+                // Generic error message
+                dispatch(
+                    addAlert<AlertInterface>({
+                        description: t("console:manage.features.actions.notification.genericError" +
+                            ".typesFetch.description"),
+                        level: AlertLevels.ERROR,
+                        message: t("console:manage.features.actions.notification.genericError.typesFetch.message")
+                    })
+                );
+            }
+        }
+    }, []);
+
+    const checkFeatureEnabledStatus = (actionType: string): boolean => {
+
+        switch (actionType) {
+            case ActionsConstants.PRE_ISSUE_ACCESS_TOKEN_URL_PATH:
+                return isFeatureEnabled(actionsFeatureConfig, "actions.filterPreIssueAccessToken");
+            case ActionsConstants.PRE_UPDATE_PASSWORD_URL_PATH:
+                return isFeatureEnabled(actionsFeatureConfig, "actions.filterPreUpdatePassword");
+            case ActionsConstants.PRE_UPDATE_PROFILE_URL_PATH:
+                return isFeatureEnabled(actionsFeatureConfig, "actions.filterPreUpdateProfile");
+            case ActionsConstants.PRE_REGISTRATION_URL_PATH:
+                return isFeatureEnabled(actionsFeatureConfig, "actions.filterPreRegistration");
+            default:
+                return false;
+        }
+    };
+
+
+    const resolveActionDescription = (): ReactNode => (
+        <>
+            { t("console:manage.features.actions.description") }
+            <DocumentationLink
+                link={
+                    getLink("develop.actions.learnMore")
+                }
+                showEmptyLink={ false }
+            >
+                { t("common:learnMore") }
+            </DocumentationLink>
+        </>
+    );
+
+    const resolveFeatureLabelClass = (featureStatus: FeatureStatusLabel): string => {
+        switch (featureStatus) {
+            case FeatureStatusLabel.BETA:
+                return "oxygen-chip-beta";
+            case FeatureStatusLabel.COMING_SOON:
+                return "oxygen-chip-coming-soon";
+        }
+    };
+
+    const resolveConfiguredLabel = (actionType: string): ReactElement => {
+        let count: number = 0;
+
+        switch (actionType) {
+            case ActionsConstants.PRE_ISSUE_ACCESS_TOKEN_URL_PATH:
+                count = typeCounts?.preIssueAccessToken;
+
+                break;
+            case ActionsConstants.PRE_UPDATE_PASSWORD_URL_PATH:
+                count = typeCounts?.preUpdatePassword;
+
+                break;
+            case ActionsConstants.PRE_UPDATE_PROFILE_URL_PATH:
+                count = typeCounts?.preUpdateProfile;
+
+                break;
+            case ActionsConstants.PRE_REGISTRATION_URL_PATH:
+                count = typeCounts?.preRegistration;
+
+                break;
+            default:
+                break;
+        }
+
+        return (
+            count > 0 ? (
+                <div className="status-tag">
```

Comment:
Can we move this out of `if` since this is common for both if and else.

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/6722/commits/5e3564e0797c09f4f20b71837b928bc9fa57c847